src/components/staffing/ProjectCell.js

```diff
@@ -2,23 +2,41 @@ import React from 'react'
 import { Cell } from 'fixed-data-table'
 
 export default class ProjectCell extends React.Component {
-
   static propTypes = {
     data: React.PropTypes.array.isRequired,
     field: React.PropTypes.string,
     rowIndex: React.PropTypes.number,
     onClick: React.PropTypes.func,
-  }
+  };
+  ignoredProjectNames = ['HOLIDAY', 'NO PROJECT NAME', 'Leave UK', 'NOTHERE'];
```

Comment:
For hard coded constants like this, the good practise is to:
- put them outside of components (either at the top of the file or in a different file like `src/constants/ignoredProjectNames.js`) 
  - why? → anything within the component takes "a little bit" of processing every time there is a re-render
- write the variable name in ALL_CAPS (with underscores instead of camel case)
  - why? → it lets devs know that this is a hardcoded constant and that it will never be modified

Reply:
Fixed!